stdlib/jdbc/src/test/resources/test-src/sql/table/table_type.bal

```diff
@@ -1423,6 +1423,40 @@ function testMultipleRowsWithForEach() returns @tainted [int, int] {
     return [rs1.INT_TYPE, rs2.INT_TYPE];
 }
 
+function testGetPrimitiveTypesWithWhileLoopAndConstructFrom() returns @tainted [int, int, float, float, boolean, string, decimal] {
+    jdbc:Client testDB = new({
+        url: "jdbc:h2:file:./target/tempdb/TEST_DATA_TABLE_H2",
```

Comment:
This type of hardcoded jdbc urls will fail this test on windows. Do you have a plan to fix this it?

Reply:
I fixed it with https://github.com/ballerina-platform/ballerina-lang/pull/18020

Reply:
This is fixed in Anupama's PR which refactors tests.